docs/general/polkadot-vision.md

```diff
@@ -0,0 +1,194 @@
+---
+id: polkadot-vision
+title: Polkadot Vision
+sidebar_label: Polkadot Vision
+description: Polkadot's Vision revealed by Gavin Wood at Decoded 2023.
+keywords: [polkadot vison, polkadot]
+slug: ../polkadot-vision
+---
+
+## A Multi-chain Future
+
+Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
+tens of thousands. Different projects have different value propositions, suggesting that the future
+will likely be multi-chain and that inter-chain communication will be crucial to establish
+collaborations and leveraging each other strengths.
+
+## Polkadot 1.0
+
+Polkadot 1.0 reflects the status of Polkadot in 2023 at time of the release of the Polkadot runtime
+v1.0. This sections focuses on Polkadot 1.0 and some philosophical digressions about network
+resilience and blockspace.
+
+Polkadot is a Layer-0 blockchain that brings to the multi-chain vision the following innovative
+concepts and initiatives:
+
+- **Application-specific Layer-1 (L1) blockchains (or parachains).** Polkadot is a sharded network
+  where data are processed in parallel within each shard. Polkadot shards can be heterogenous (i.e.
+  they do not need to be the exact copy of the original shards, like in Ethereum shards). This
+  allows to build L1 chains designed explicitly around their application and value proposition.
+- **Shared security and financial scalability of L1 chains**. Any L1 chain attached to a Polkadot
+  core (and any other L2 solution to scale that L1) will benefit from Polkadot shared security
+  model. This means the Polkadot Nominated-Proof-of-Stake (NPoS) mechanism will secure L1 and L2
+  chains without them worrying about having enough proof-of-stake.
```

Comment:
L2 security assumptions differ from parachain security assumptions. By design. all the parachains enjoy the same levels of security guarantees provided by Polkadot. The L2 solutions developed on top of L1s do not, as they do not adhere to any particular standard.

```suggestion
- **Shared security and financial scalability of L1 chains**. Any L1 chain attached to a Polkadot
  core can benefit from Polkadot shared security
  model. This means the Polkadot Nominated-Proof-of-Stake (NPoS) mechanism along with its consensus mechanism, will secure L1
  chains out-of-the-box without having to bootstrap security on their own.
```